wallet/src/Pos/Wallet/Web/Mode.hs

```diff
@@ -3,13 +3,13 @@
 {-# LANGUAGE TypeOperators       #-}
 
 module Pos.Wallet.Web.Mode
-       ( WalletWebMode
+       ( WalletMempoolExt
```

Comment:
tbh I would like you to remove this export because it may cause dependence of other modules on `Pos.Wallet.Web.Mode` (by mistake or if you aren't familiar with code pretty well). It's better to leave it only in `WalletMode.hs`

Reply:
Right